### PR TITLE
Add a check for fileMeta.size

### PR DESCRIPTION
**Pull request by shilwantk**
## Summary
Added a check for nil values of filemeta.size

### CODE REVIEW

File: Sources/Utilities/ALMessage+Extension.swift [hunk 1]
```diff
@@ -265,10 +265,9 @@ extension ALMessage {
     }
 
     var size: Int64 {
-        guard let fileMeta = fileMeta, let size = Int64(fileMeta.size) else {
-            return 0
-        }
-        return size
+        guard let fileMeta = fileMeta, let size = fileMeta.size else { return 0 }
+        guard let sizeInt = Int64(size) else { return 0 }
```
Comment (mukeshthawani) [inline Sources/Utilities/ALMessage+Extension.swift]: We can put all the checks in a single guard statement.

**Review by mukeshthawani — APPROVED** [review]

File: Sources/Utilities/ALMessage+Extension.swift (hunk 1 above)
Comment (shilwantk) [reply Sources/Utilities/ALMessage+Extension.swift]: Done

**Comment by mukeshthawani** [comment]
@shilwantk We can merge it and use in the latest version of Agent app.

**Comment by shilwantk** [comment]
> @shilwantk We can merge it and use in the latest version of Agent app.

Okay

**Pull request merged**